apps/condo/domains/billing/schema/BillingReceiptFile.js

```diff
@@ -107,9 +117,9 @@ const BillingReceiptFile = new GQLListSchema('BillingReceiptFile', {
             knexOptions: { isNotNullable: true }, // Required relationship only!
             kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
             hooks: {
-                resolveInput: async ({ resolvedData, operation, context }) => {
+                resolveInput: async ({ resolvedData, operation }) => {
```

Comment:
Do not update file if receipt already exists 
Cases when we need to update PDF - will be resolved manually

Reply:
I'll implement such logic in RegisterBillingReceiptFileService